feat: refactor truncate string package (#1097)
stdlib-js/stdlib: lib/node_modules/@stdlib/string/base/truncate-code-points/README.md

<!--

@license Apache-2.0

Copyright (c) 2023 The Stdlib Authors.

Licensed under the Apache License, Version 2.0 (the "License");
you may not use this file except in compliance with the License.
You may obtain a copy of the License at

   http://www.apache.org/licenses/LICENSE-2.0

Unless required by applicable law or agreed to in writing, software
distributed under the License is distributed on an "AS IS" BASIS,
WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
See the License for the specific language governing permissions and
limitations under the License.

-->

# truncateCodePoints

> Truncate the Unicode code points of a provided string in order to return a string having a specified length.

<section class="usage">

## Usage

```javascript
var truncateCodePoints = require( '@stdlib/string/base/truncate-code-points' );
```

#### truncateCodePoints( str, len, ending )

Truncates the code points of a provided string in order to return a string having a specified length.

```javascript
var out = truncateCodePoints( 'beep boop', 7, '...' );
// returns 'beep...'

> var out = truncateCodePoints( 'beep boop', 7, '!' );

Reviewer: ```suggestion

out = truncateCodePoints( 'beep boop', 7, '!' );
```

<!--

@license Apache-2.0

Copyright (c) 2023 The Stdlib Authors.

Licensed under the Apache License, Version 2.0 (the "License");
you may not use this file except in compliance with the License.
You may obtain a copy of the License at

   http://www.apache.org/licenses/LICENSE-2.0

Unless required by applicable law or agreed to in writing, software
distributed under the License is distributed on an "AS IS" BASIS,
WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
See the License for the specific language governing permissions and
limitations under the License.

-->

# truncateCodePoints

> Truncate the Unicode code points of a provided string in order to return a string having a specified length.

<section class="usage">

## Usage

```javascript
var truncateCodePoints = require( '@stdlib/string/base/truncate-code-points' );
```

> #### truncateCodePoints( str, len, ending )

Reviewer: ```suggestion
#### truncateCodePoints( str, N, ending )
```

> #### truncateCodePoints( str, N, ending )

Reviewer: ```suggestion
#### truncateCodePoints( str, len, ending )
```

Updating to match documentation in other files.

> Truncates the code points of a provided string in order to return a string having a specified length.

Reviewer: ```suggestion
Truncates the Unicode code points of a provided string in order to return a string having a specified number of Unicode code points.
```

<!--

@license Apache-2.0

Copyright (c) 2023 The Stdlib Authors.

Licensed under the Apache License, Version 2.0 (the "License");
you may not use this file except in compliance with the License.
You may obtain a copy of the License at

   http://www.apache.org/licenses/LICENSE-2.0

Unless required by applicable law or agreed to in writing, software
distributed under the License is distributed on an "AS IS" BASIS,
WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
See the License for the specific language governing permissions and
limitations under the License.

-->

# truncateCodePoints

> > Truncate code points of a string to a specified length.

Reviewer: Same general comment as elsewhere. Would you be willing to update the descriptions here and elsewhere to something like

```suggestion
> Truncate the code points of a provided string in order to return a string having a specified length.
```

> > Truncate the code points of a provided string in order to return a string having a specified length.

Reviewer: ```suggestion
> Truncate the Unicode code points of a provided string in order to return a string having a specified length.
```

Here and elsewhere.

<section class="usage">

## Usage

```javascript
var truncateCodePoints = require( '@stdlib/string/base/truncate-code-points' );
```

> #### truncateCodePoints( str, len\[, ending] )

Reviewer: Similar to other "base" packages, we try to avoid variadic interfaces. Can you make `ending` a required argument here and elsewhere?

Truncates the code points of a provided string in order to return a string having a specified length.

```javascript
var out = truncateCodePoints( 'beep boop', 7, '...' );
// returns 'beep...'

> ```
> 
> By default, the truncated string is appended with `'...'`. To customize the truncated string, provide an `ending` argument:
> 
> ```javascript

Reviewer: ```suggestion

```

> By default, the truncated string is appended with `'...'`. To customize the truncated string, provide an `ending` argument:
> 
> ```javascript
> var out = truncateCodePoints( 'beep boop', 7, '!' );
> // returns 'beep b!'
> 
> out = truncateCodePoints( 'beep boop', 7, '!!!' );
> // returns 'beep!!!'
> ```
> 

Reviewer: ```suggestion
```
